examples/jsm/controls/OrbitControls.d.ts

```diff
@@ -44,7 +44,89 @@ export class OrbitControls {
 
 	enableKeys: boolean;
 	keys: { LEFT: number; UP: number; RIGHT: number; BOTTOM: number; };
-	mouseButtons: { LEFT: MOUSE; MIDDLE: MOUSE; RIGHT: MOUSE; };
+	mouseButtons: { LEFT: MOUSE; MIDDLE: MOUSE; RIGHT: MOUSE;  };
```

Comment:
I don't think this style is good. I would prefer to say what key is assigned to what action. E.g.

```
controls.mouseButtons.ROTATE = THREE.MOUSE.LEFT;
```

@mrdoob How do you feel about this?